src/helpers/files.ts

```diff
@@ -184,24 +179,16 @@ const EMPTY_STRING = '' as const
 
 const isNegated = (path: string) => path.startsWith('!')
 
-/**
- *
- * @param {string} projectRoot
- * @param {string[]} coverageFilePatterns
- * @returns {Promise<string[]>}
- */
 export async function getCoverageFiles(
   projectRoot: string,
   coverageFilePatterns: string[],
 ): Promise<string[]> {
-  const globstar = (pattern: string) => `**/${pattern}`
-
   return glob(coverageFilePatterns.map((pattern: string) => {
     const parts = []
 
     if (isNegated(pattern)) {
       parts.push('!')
-      parts.push(globstar(pattern.substr(1)))
+      parts.push(globstar(pattern.substring(1)))
```

Comment:
`substr` was marked as deprecated. `substring` was the suggested alternative to use.